programming/vision-tracking.md

```diff
@@ -0,0 +1,42 @@
+# Basics of Vision Tracking
+Vision Tracking!!! It allows the robot to see. It likes shiny tape. You need a camera, raspberry pi, LED ring, and something to vision track.
```

Comment:
We will be doing vision tracking onboard the RoboRIO, so a Raspberry Pi is not necessary.